app/views/layouts/application.html.erb

```diff
@@ -19,6 +20,7 @@
     window.mumukiLocale = <%= raw Organization.locale_json %>;
   </script>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <script type="text/javascript" src="<%= extension_javascript_url %>" defer></script>
   <%= login_form.header_html %>
```

Comment:
Nice :smile: ! I din't know about that `defer` attribute. Is it widely supported by browsers?